circleguard/gui/main_tab.py

```diff
@@ -563,6 +563,11 @@ def _remove_replay(replay):
                         _check_event(event)
                         # skip replays which have no map info
                         if cg.map_available(replay):
+                            # TODO: cache hits so they don't need to be calculated multiple times for a UR investigation
```

Comment:
hits calculation is on the order of 0.01 seconds once the replay has been loaded and the beatmap cached, so it's probably not that big deal for us because the vast majority of wait time is spent on ratelimits and beatmap downloading. It would still be good to cache hits in circlecore for other applications which don't have those limitations, however.